test/extended/cli/mustgather.go

```diff
@@ -30,6 +30,14 @@ var _ = g.Describe("[sig-cli] oc adm must-gather", func() {
 
 	oc := exutil.NewCLI("oc-adm-must-gather").AsAdmin()
 
+	g.BeforeEach(func() {
+		isMicroShift, err := exutil.IsMicroShiftCluster(oc)
```

Comment:
must-gather uses many OCP apigroups not present in microshift, is it better to annotate the tests instead?

Reply:
Yes, that can be a better approach. I've added `[apigroup:config.openshift.io]` in these tests, please let me know if this looks good. Thanks!